docs/en/observability/uptime-intro.asciidoc

```diff
@@ -1,20 +1,25 @@
-[[monitor-uptime-synthetics]]
-= Uptime and synthetic monitoring
-
-++++
-<titleabbrev>Uptime and synthetic monitoring</titleabbrev>
-++++
+[[uptime-intro]]
+= Uptime
 
 [[uptime-monitors]]
 
-The {uptime-app} uses {agent} to periodically check the status of your services and applications.
-Monitor the availability of network endpoints and services using the following Uptime monitors:
+[IMPORTANT]
+====
+The {uptime-app} is for viewing result data from lightweight monitors running through
+{heartbeat} and <<uptime-get-started,configured with a traditional `heartbeat.yml` file>>.
+This is for TCP, HTTP or ICMP monitors that you have configured and run from your own
+infrastructure with {heartbeat} natively.
 
-* <<monitoring-uptime>>
-* <<monitoring-synthetics>>
+For browser-based monitors, a richer management and reporting experience,
+and more capabilities such as triaging and responding to alerts, use the
+<<monitor-uptime-synthetics,Synthetics app>> instead of the {uptime-app}.
+====
```

Comment:
@paulb-elastic @drewpost @florent-leborgne what do you think about this language on the [Uptime intro page](https://observability-docs_2593.docs-preview.app.elstc.co/guide/en/observability/master/uptime-intro.html) to explain the relationship between Uptime and Synthetics?

Reply:
LGTM